test/stdcpp/src/array.cpp

```diff
@@ -0,0 +1,17 @@
+#include <array>
+
+int fromC_val(std::array<int, 5>);
+int fromC_ref(const std::array<int, 5>&);
+
+int sumOfElements_ref(const std::array<int, 5>& arr)
+{
+    int r = 0;
+    for (size_t i = 0; i < arr.size(); ++i)
+        r += arr[i];
+    return r;
+}
+
+int sumOfElements_val(std::array<int, 5> arr)
+{
+    return sumOfElements_ref(arr) + fromC_ref(arr) + fromC_val(arr);
+}
```

Comment:
I'd also like to see a function which returns an `std::array` by value and by `ref`.